sdk/storage/AzureStorageBlob/AzureStorageBlob.xcodeproj/project.pbxproj

```diff
@@ -76,6 +78,8 @@
 		0A2868B423FF38AA0084CF15 /* MultiBlobTransfer+CoreDataProperties.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = "MultiBlobTransfer+CoreDataProperties.swift"; path = "/Users/travisprescott/repos/azure-sdk-for-ios/sdk/storage/AzureStorageBlob/Source/Data/MultiBlobTransfer+CoreDataProperties.swift"; sourceTree = "<absolute>"; };
 		0A2868B523FF38AA0084CF15 /* BlobTransfer+CoreDataProperties.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = "BlobTransfer+CoreDataProperties.swift"; path = "/Users/travisprescott/repos/azure-sdk-for-ios/sdk/storage/AzureStorageBlob/Source/Data/BlobTransfer+CoreDataProperties.swift"; sourceTree = "<absolute>"; };
 		0A2868B623FF38AA0084CF15 /* BlockTransfer+CoreDataProperties.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = "BlockTransfer+CoreDataProperties.swift"; path = "/Users/travisprescott/repos/azure-sdk-for-ios/sdk/storage/AzureStorageBlob/Source/Data/BlockTransfer+CoreDataProperties.swift"; sourceTree = "<absolute>"; };
```

Comment:
Fixed. Since it was just these three files that had this issue, it looks like it happens when you use the "Create NSManagedObject Subclass..." wizard in the IDE, and there are no obvious options to affect this. Removing the file references and re-adding them fixed the problem.